### PR TITLE
feat(docs): add pyth to kaia docs

### PR DESCRIPTION
**Pull request by mariobern**
## Proposed changes

Hi all,

We'd like to get featured in Kaia documentation as one of the leading oracles supporting the ecosystem.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Minor Issues and Typos
- [x] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/kaiachain/kaia-docs/blob/main/CONTRIBUTING.md)
- [x] I have read the [CLA](https://gist.github.com/kaiachain-dev/bbf65cc330275c057463c4c94ce787a6) and signed by comment ```I have read the CLA Document and I hereby sign the CLA``` in first time contribute
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Related issues

NA

## Further comments

NA

### CODE REVIEW

**Comment by ayo-klaytn** [comment]
Hi @mariobern Thanks for your contribution.

In the meantime, i'll like you to sign the [CLA](https://gist.github.com/kaiachain-dev/bbf65cc330275c057463c4c94ce787a6) as suggested by the bot.

If you have done that, kindly comment `recheck` in this PR.

**Comment by mariobern** [comment]
recheck

**Comment by mariobern** [comment]
`recheck`

**Comment by mariobern** [comment]
@ayo-klaytn hi Ayo, seems like `recheck` isn't working

**Comment by ayo-klaytn** [comment]
Hi @mariobern 

Kindly comment:   I have read the CLA Document and I hereby sign the CLA   in this PR.
Then comment : recheck  afterwards.

You can use this [PR](https://github.com/kaiachain/kaia-docs/pull/134) for reference

**Comment by mariobern** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mariobern** [comment]
`recheck`

**Comment by mariobern** [comment]
much appreciated @ayo-klaytn for the help!

**Comment by mariobern** [comment]
still missing the banner but everything else is good

File: docs/build/tools/oracles/pyth-network.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Pyth Network
+
+// ![](/img/banners/kaia-pyth.png)
```
[the remark this answers is not in the document]
Comment (ayo-klaytn) [reply docs/build/tools/oracles/pyth-network.md:3]: @mariobern kindly update the image with the banner below. Thanks 

kaia-pyth-banner

**Comment by mariobern** [comment]
@ayo-klaytn hi I'm trying to find the folder img/banners but can't seem to find it anywhere in the repo, could you point me to it?

**Comment by scott-klaytn** [comment]
> @ayo-klaytn hi I'm trying to find the folder img/banners but can't seem to find it anywhere in the repo, could you point me to it?

스크린샷 2024-10-30 오전 8 44 40

**Comment by mariobern** [comment]
@ayo-klaytn @scott-klaytn done, kindly review and let me know if I miss anything

**Review by ayo-klaytn — APPROVED** [review]

**Review by scott-klaytn — APPROVED** [review]

**Comment by ayo-klaytn** [comment]
Hi @mariobern 

Just merged the PR. Thanks for your contribution 🫡

**Pull request merged**